### ISSUE TITLE
Dev To Expand on Your Idea

### ISSUE THREAD

**Issue by Boolyman** [LothrazarMinecraftMods/OceanFloorControl#2]
Hello there.  I have a mod dev that is working with me on an upcoming project.  We are trying to find a way to get clusters of configurable blocks to spawn on the ocean floor.  Much like Glowstone spawns on the Nether ceiling, these clusters would spawn on the top of the ocean floor.  This would force players to go into the water, use scooba gear, etc... in order to get these clusters.  

Long story short, in figuring out a way to code this, my dev associate viewed your code for making clay spawn on the ocean floor.  Though we would need to make a different code in order to have clusters that spawn on top of the ocean floor, the "spirit" of the mod and its code would highly resemble yours.  That being said we wanted to reach out to you and get your blessing on this project, just to be 100% transparent.  

**TL;DR** We are making a similarly coded mod that spawns clusters of configurable ores/blocks on the ocean floor.  We want to make you aware and see if you have any objections.

**Comment by Lothrazar** [comment]
Yeah for sure, you can totally use some of this code.  I basically just learned it from random tutorials and blogs and code samples like that.   This is why i made it open source.

But thanks for asking! Also let me know when the mod is done so i can download it.  ill follow you on twitter too if you use that for mod updates, seems like a lot of people do.
